charts/buffer/values.yaml

```diff
@@ -16,7 +16,7 @@ appVersion:
 
 imageConfig:
   # imageConfig.repository -- Image repository
-  repository: gcr.io/terra-kernel-k8s/terra-buffer
+  repository: gcr.io/terra-kernel-k8s/terra-resource-buffer
```

Comment:
LGTM!

I have a slight preference for renaming the docker image to `buffer`, which matches the conventions the other Terra apps use (Cromwell is called `cromwell`, Leonardo is called `leonardo`, etc). I'm not sure why the image needs to be qualified with `terra-`, when it's already deployed to a repository that includes the name `terra`.

Reply:
I think the convention is to use the github repo name:
https://github.com/DataBiosphere/terra-resource-buffer/blob/master/.github/workflows/master_push.yml#L74
And I think that is what janitor, WM and RBS did. And I think @gjuggler mentioned somewhere that the new services repo name should starts with `terra-`